### PR TITLE
onboarding page

### PR DESCRIPTION
**Pull request by maelle**
cf https://github.com/ropensci/onboarding-meta/issues/16

Also note that in its current state this PR creates a onboarding/ page which conflicts with the current URL for the volunteering from. And anyway we might want to call it peer-review cf https://github.com/ropensci/onboarding-meta/issues/11

### CODE REVIEW

**Comment by maelle** [comment]
it's a boring page, but I think that with minimal efforts we can make it ok at least as a place-holder.

Further improvements could include embedding the gitbook. That probably doesn't work, but even just having a clickable cover for the gitbook would be huge.

**Comment by maelle** [comment]
Direct link to the preview https://deploy-preview-332--ropensci.netlify.com/onboarding/

File: content/software-review.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
++++
+title = "rOpenSci software review"
+description = "rOpenSci's open peer-review system of R packages contributed by the community"
++++
+
+rOpenSci's [suite of packages](/packages/) is partly contributed by staff members and partly contributed by community members, which means the suite stems from a great diversity of skills and experience of developers. How to ensure quality for the whole set? That's where our software system comes into play: packages contributed by the community undergo a **transparent, constructive, non adversarial and open review process**. For that process relying mostly on **volunteer work**, _[associate editors](/about#team)_ manage the incoming flow and ensure progress of submissions; _authors_ create, submit and improve their package; _[reviewers](https://github.com/ropensci/onboarding#reviewers)_, two per submission, examine the software code and user experience.
```
Comment (karthik) [inline content/software-review.md]: > rOpenSci’s suite of packages is partly contributed by staff members and partly contributed by community members

rOpenSci’s suite of packages is comprised of contributions from staff engineers and the wider R community.

Comment (karthik) [reply content/software-review.md]: > which means the suite stems from a great diversity of skills and experience of developers.

bringing considerable diversity of skills, expertise and experience to bear on the suite.

Comment (karthik) [reply content/software-review.md]: >  How to ensure quality for the whole set?

We don't evaluate the whole set. We do it by package. So you can say: How do we ensure that every package is held to a high standard?

**Comment by maelle** [comment]
I should replace the `ropenaq` example. Maybe `bowerbird`'s thread, but its reviews were abnormally long.

**Comment by karthik** [comment]
I think `ropenaq` is a great example. Another one that is shorter and easier to grasp might be the April Wright one that was just finished. Either way this looks good to merge from my side.

**Pull request merged**